packages/clerk-js/src/ui/components/UserProfile/UserProfileBlockButtons.tsx

```diff
@@ -15,7 +15,7 @@ export const BlockButton = (props: BlockButtonProps) => {
 };
 
 export const AddBlockButton = (props: BlockButtonProps) => {
-  const { id, ...rest } = props;
+  const { id, icon, ...rest } = props;
```

Comment:
If we're going to be passing the icon, shouldn't we just name this `BlockButton` ?